src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/NodeBuilder.cs

```diff
@@ -41,13 +41,25 @@ public NodeBuilder(string rootFolder)
         public static NodeBuilder Create(object caller, [CallerMemberName] string callingMethod = null)
         {
             string testFolderPath = TestBase.CreateTestDir(caller, callingMethod);
-            return new NodeBuilder(testFolderPath);
+            return CreateNodeBuilder(testFolderPath);
         }
 
         public static NodeBuilder Create(string testDirectory)
         {
             string testFolderPath = TestBase.CreateTestDir(testDirectory);
-            return new NodeBuilder(testFolderPath);
+            return CreateNodeBuilder(testFolderPath);
+        }
+
+        /// <summary>
+        /// Creates a node builder instance and disable logs.
+        /// To enable logs look the <see cref="WithLogsEnabled"/>  method.
+        /// </summary>
+        /// <param name="testFolderPath">The test folder path.</param>
+        /// <returns>A NodeBuilder instance.</returns>
```

Comment:
Probably can remove this line. Or change to `A <see cref="NodeBuilder"/> instance with logs disabled.`